src/canopy-webpack-config.js

```diff
@@ -62,7 +62,7 @@ module.exports = function(name, overridesConfig) {
         ],
       },
       plugins: [
-        new CleanWebpackPlugin(['build']),
```

Comment:
clean-webpack-plugin now defaults to cleaning webpack files from the path defined in `output.path`. This little bit of configuration is no longer required for our usage.